src/ssl.c

```diff
@@ -2829,6 +2829,60 @@ const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbyname(const char *name)
     return NULL;
 }
 
+const WOLFSSL_EVP_CIPHER *wolfSSL_EVP_get_cipherbynid(int id)
+{
+    WOLFSSL_ENTER("EVP_get_cipherbynid");
+
+    switch(id) {
+
+#ifndef NO_AES
+        case NID_aes_128_cbc:
+            return wolfSSL_EVP_aes_128_cbc();
+        case NID_aes_192_cbc:
+            return wolfSSL_EVP_aes_192_cbc();
+        case NID_aes_256_cbc:
+            return wolfSSL_EVP_aes_256_cbc();
+#if defined(OPENSSL_EXTRA)
```

Comment:
Yes that would work for the AES NID values, but also be sure to guard NID values other then the AES ones too. All NID values from wolfssl/openssl/* should be guarded with an OPENSSL_EXTRA macro when used in files like src/ssl.c.